### PR TITLE
Oauth helm chart

### LINKED ISSUE #9: OAuth2 Helmchart Spec
As a chart developer, I want to provide a deployment package for any k8s cluster, to [1]benefiting the operations, and to [2]promoting the Client Agent SSO Authentication.

### PR DESCRIPTION
**Pull request by ramaraosrikakulapu**
PR is to create the helm chart for oAuth package and requirement is part of following features -
- [DTEC-155](https://ge-dw.aha.io/features/DTEC-155) 
- [DTEC-142](https://ge-dw.aha.io/features/DTEC-142)

### CODE REVIEW

**Comment by ramaraosrikakulapu** [comment]
This is working branch as of 03/10.

**Comment by ayasuda-ge** [comment]
@ramaraosrikakulapu please help in the clean up-

Remove:
k8s/pkg/*

File: scripts/k8s-deployment-sim.sh [hunk 1]
```diff
@@ -66,17 +66,16 @@ kubectl describe services $(kubectl get services|grep agent-plg|awk '{print $1}'
 printf "\n\n\n*** [3.4] clear installation\n"
 kubectl delete --all deployments && kubectl delete --all pods && kubectl delete --all services
 
-printf "\n\n\n*** [4] install lber in k8s\n"
-#helm install k8s/agent+lber --generate-name
-: 'printf "\n\n\n*** [4.1] verify installation\n"
+printf "\n\n\n*** [4] install oauth in k8s\n"
+helm install k8s/examples/oauth --set-file global.oauthConfig=k8s/examples/oauth/oauth.env --generate-name
```
Comment (ayasuda-ge) [inline scripts/k8s-deployment-sim.sh:70]: No pvc test available. Please verify in minikube @ramaraosrikakulapu

File: scripts/k8s-deployment-sim.sh [hunk 2]
```diff
@@ -66,17 +66,16 @@ kubectl describe services $(kubectl get services|grep agent-plg|awk '{print $1}'
...
+printf "\n\n\n*** [4.1] verify installation\n"
 kubectl get deployments && kubectl get pods && kubectl get services && kubectl get ingresses
 printf "\n\n\n*** [4.2] verify deployment spec\n"
-kubectl describe deployments $(kubectl get deployments|grep agent-plg|awk '{print $1}'|head -n 1)
-printf "\n\n\n*** [4.2] verify statefulset spec\n"
-kubectl describe sts $(kubectl get sts|grep agent-plg|awk '{print $1}'|head -n 1)
+kubectl describe deployments $(kubectl get deployments|grep oauth|awk '{print $1}'|head -n 1)
 printf "\n\n\n*** [4.3] verify service spec\n"
-kubectl describe services $(kubectl get services|grep agent-plg|awk '{print $1}'|head -n 1)
+kubectl describe services $(kubectl get services|grep oauth|awk '{print $1}'|head -n 1)
```
Comment (ayasuda-ge) [inline scripts/k8s-deployment-sim.sh:76]: please describe pvc in minikube @ramaraosrikakulapu

File: scripts/k8s-deployment-sim.sh (hunk 1 above)
Comment (ramaraosrikakulapu) [reply scripts/k8s-deployment-sim.sh:70]: pvc tests are added now.

File: scripts/k8s-deployment-sim.sh (hunk 2 above)
Comment (ramaraosrikakulapu) [reply scripts/k8s-deployment-sim.sh:76]: pvc tests are added now

**Comment by ramaraosrikakulapu** [comment]
> @ramaraosrikakulapu please help in the clean up-
> 
> Remove:
> k8s/pkg/*

fixed

**Comment by ramaraosrikakulapu** [comment]
@ayasuda-ge .. can you please take a look at PR when possible and please merge, if every thing looks good.

**Comment by ramaraosrikakulapu** [comment]
@ayasuda-ge .. as suggested I updated the helm versions for all packages in CI.

**Review by ayasuda-ge — APPROVED** [review]
pvc looks good!

**Pull request merged**